extensions/ql-vscode/test/unit-tests/common/files.test.ts

```diff
@@ -580,4 +572,23 @@ describe("findCommonParentDir", () => {
 
     expect(commonDir).toEqualPath(rootDir);
   });
+
+  it("should return the parent dir of a single file", async () => {
+    const dataDir = join(__dirname, "../../data");
```

Comment:
Hmm, this works, but there is a non-zero chance it'll cause confusing errors for us in the future. The `extensions/ql-vscode/test/data` directory and `extensions/ql-vscode/test/data/query.ql` file aren't really related to this test, so someone could easily move them and not realise it'll affect these tests.

I suppose maybe it's ok because if those files are moved then these tests will immediately start failing by calling `lstatSync` on a path that doesn't exist. So maybe it's fine, but I wanted to mention it.

Reply:
There are many places where we do this so this wouldn't be the only test that would be affected. It's not ideal, but that's how we've done things so far.